packages/liferay-js-insights/reporters/table.js

```diff
@@ -6,23 +6,36 @@
 
 const Table = require('cli-table');
 
+const DEFAULT_CONFIG = {
+	output: 'app,name,dependencies.clay3,dependencies.react'
+};
+
+/**
+ * Traverses an object provided a path to a property and returns its value if found or null if the path can't be reached.
+ */
+const idx = (p, o) => p.reduce((xs, x) => (xs && xs[x] ? xs[x] : null), o);
```

Comment:
Bet you can't wait for https://github.com/tc39/proposal-optional-chaining 😂

Reply:
Good things worth waiting for ;)